examples/map/compare_rotation_results.py

```diff
@@ -0,0 +1,47 @@
+# # Comparing Results of `map.rotate` Between Library Implementations
+# ----------
+
+import sunpy.data.sample
+import sunpy.map
+import numbers
+import numpy as np
+import cv2
+
+# Rotating a map in Sunpy has a choice between three libraries: `skimage', `scipy`, and `cv2`.
+# One can also create a custom rotation function and drop it into `map.rotate`.
+# Since all these options use different algorithms, we need a basis for comparison:
+# what is an acceptable margin of difference in the final, rotated data product?
+# One option is to use the Symmetric Mean Absolute Percentage Error
+# (https://en.wikipedia.org/wiki/Symmetric_mean_absolute_percentage_error;
+# the third equation in the link). This is a bit rudimentary, since it looks
+# at all of the pixels in the image (and not just the solar disk), but it works
+# for a quick check.
+
+# SymmetricMeanAbsolutePercentageError
+# returns between 0%-100% of average error between arr1 and arr2
```

Comment:
These two lines should be moved to a docstring of the function using a multi-line `"""` comment.